_posts/2020-03-04-arc-limitations.adoc

```diff
@@ -91,20 +97,21 @@ On the other hand, the API is quite complex the implementation must provide non-
   
 Reasons to skip:
 
-* Portable extensions are inherently _runtime components_ that are very often stateful. 
-+ 
+* Noticable performance overhead,
+* Portable extensions are inherently _runtime components_ that are very often stateful.
```

Comment:
It might be worth expanding on on why are they runtime components as I think it's one of the most common questions and having a detailed explanation wouldn't hurt here.